### ISSUE TITLE
[css] duplicate classes in selectors

### ISSUE THREAD

**Issue by mvorisek** [fomantic/Fomantic-UI#2638]
# Bug Report

in nightly `semantic.css` there are selectors like:

```
  .ui.ui.ui.ui.table:not(.unstackable) > tr > th,
  .ui.ui.ui.ui.table:not(.unstackable) > thead > tr > th,
  .ui.ui.ui.ui.table:not(.unstackable) > tbody > tr > th,
  .ui.ui.ui.ui.table:not(.unstackable) > tfoot > tr > th,
  .ui.ui.ui.ui.table:not(.unstackable) > tr > td,
  .ui.ui.ui.ui.table:not(.unstackable) > tbody > tr > td,
  .ui.ui.ui.ui.table:not(.unstackable) > tfoot > tr > td {
    background: none;
    border: none;
    padding: 0.25em 0.75em;
    box-shadow: none;
  }
```

## Expected result

class is enough to be mentioned only once per selector element

present also in `semantic.min.css` making the file larger than needed

**Comment by lubber-de** [comment]
This is intentional as the multiple `ui` fixes specificity to avoid using `!important` at several places

**Comment by lubber-de** [comment]
Duplicate of #1630

**Comment by mvorisek** [comment]
I did not know that, thanks.

https://stackoverflow.com/questions/11572081/css-class-repetition-to-increase-specificity
http://jsfiddle.net/1ew38yhj/1/

**Comment by mvorisek** [comment]
there is also:

```
.ui.basic.buttons .button:active {
  box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.15) inset, 0 1px 4px 0 rgba(34, 36, 38, 0.15) inset inset;
}
```

is the `inset` repeated on the end twice intended?

**Comment by lubber-de** [comment]
> there is also:
> 
> ```
> .ui.basic.buttons .button:active {
>   box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.15) inset, 0 1px 4px 0 rgba(34, 36, 38, 0.15) inset inset;
> }
> ```
> 
> is the `inset` repeated on the end twice intended?
> 
> also:
> 
> ```
> .ui.button:active,
> .ui.active.button:active {
>   background-color: #babbbc;
>   background-image: "";
>   color: rgba(0, 0, 0, 0.9);
>   box-shadow: 0 0 0 1px transparent inset, none;
> }
> ```

Fixed by #2639 

> 
> also according to the [jigsaw.w3.org/css-validator/validator](https://jigsaw.w3.org/css-validator/validator) some properties are "cleared" with `""`, which is not officially valid value to do so

This is something we would need to fix via post css, as the empty strings are invalid and wont really reset the property.
Setting them was the attempt by original SUI to still be able to support the related LESS variable (so other themes can set some valid values) but the default theme does not set anyting to satisfy the LESS parser.

> 
> also some icons are defined twice:
> 
> ```
> i.icon.in.cart::before {
>   content: "\f218";
> }
> ...
> i.icon.in.cart::before {
>   content: "\f218";
> }
> ```

Fixed by #2640

**Comment by mvorisek** [comment]
I have rechecked the latest/2.9.1 css with https://jigsaw.w3.org/css-validator/validator and there are still some errors:

![image](https://user-images.githubusercontent.com/2228672/213179525-d1e1b51e-4e0d-4990-b165-07cd9e5af9a2.png)

![image](https://user-images.githubusercontent.com/2228672/213179555-4a2fcdac-ee70-4cc3-a951-11bbec108cd9.png)

![image](https://user-images.githubusercontent.com/2228672/213179590-1f19c12b-7acc-4eaa-99b7-8275cf6911a4.png)

![image](https://user-images.githubusercontent.com/2228672/213179629-83e55677-d1c4-4a39-aef3-5e2914b132a3.png)

![image](https://user-images.githubusercontent.com/2228672/213179700-0873e97d-32f4-48d0-8f61-dbd0bdd38167.png)

![image](https://user-images.githubusercontent.com/2228672/213179766-c46afba4-38f4-4881-985f-00e0cdec85ed.png)

![image](https://user-images.githubusercontent.com/2228672/213179788-9cc3a769-a6ee-4076-9349-7861610a2038.png)
